botocore/endpoint.py

```diff
@@ -38,6 +38,20 @@
 filter_ssl_warnings()
 
 
+def create_endpoint_url(endpoint_config, is_secure):
```

Comment:
I'll move this method, and remove endpoint related code from the endpoint module.